test/fixtures/sign-requests/post_sign_requests_id_resend.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Do we need this file? Can the test be written without it (resend endpoint returns only status code)